CHANGELOG.md

```diff
@@ -243,6 +243,13 @@ Reflection
   outside of the context that the `TC` computation is invoked from
   anymore. The telescope is now relative to that context instead.
 
+* `FOREIGN` and `COMPILE` pragmas can now be generated using two new reflection primitives:
+
+  ```agda
+  pragmaForeign : String → String → TC ⊤
+  pragmaCompile : String → Name → String → TC ⊤
+  ```
+
```

Comment:
This PR won't likely go on 2.6.3.  Because we are still in the limbo state between 2.6.3 and 2.6.4, the CHANGELOG cannot be used for 2.6.4 features yet.  Maybe this is better placed in `doc/release-notes/2.6.4.md `.